### PR TITLE
feat: created favourites page

### PR DESCRIPTION
**Pull request by vaibhav-mahalle**
- added cards to the favorites page
- added the favorite icon and rating to card

### CODE REVIEW

File: pages/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+            <div class="cart-input flex-row">
+                <input type="text" placeholder="search">
+                <a class="btn-link-noBorder" href="#">   
+                    <span class="material-icons">search</span>
+                </a>
+            </div>
+            <div class="button-box flex-row">
+              
+              <div class="box-pill">
+                <a class="btn btn-link-noBorder" href="">
```
Comment (shuntzhere) [inline pages/wishlist.html:30]: Don't leave blank href.

File: pages/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+                   <h3 class="font-md font-medium">Behrouz</h3>
+                   <h5 class="font-sm font-light p-b-1">Biryani</h5>
+                 </div>
+                   
+                   <div class="card-badge card-badge-position">
+                     Promoted
+                   </div>
+
+                   <div class="favourite-icon favourite-icon-position">
+                    <a class="btn btn-link-noBorder" href="">
```
Comment (shuntzhere) [inline pages/wishlist.html:96]: Don't leave blank hrefs

**Review by shuntzhere — APPROVED** [review]
Code looks good. Suggested a few changes do give a look.

File: pages/wishlist.html [hunk 3]
```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link
+      rel="stylesheet"
+      href="https://dexterous-ui.netlify.app/stylesheets/main.css"
+    />
```
Comment (tanishq20) [inline pages/wishlist.html:9]: You can import this directly in main CSS file.
By this you will not have to link it in every HTML file.

File: pages/wishlist.html [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+                   <h3 class="font-md font-medium">Chitale</h3>
+                   <h5 class="font-sm font-light p-b-1">Ice cream</h5>
+                 </div>
+                   
+                   <div class="card-badge card-badge-position">
+                     Bestseller
+                   </div>
+
+                   <div class="favourite-icon favourite-icon-position">
+                    <a class="btn btn-link-noBorder" href="">
```
Comment (tanishq20) [inline pages/wishlist.html:68]: Here href is blank

**Review by tanishq20 — COMMENTED** [review]
Code looks clean. Suggested minor changes.

**Pull request merged**